web/scm/src/app/services/gapi/GoogleApiConfig.ts

```diff
@@ -0,0 +1,17 @@
+import ClientConfig = gapi.auth2.ClientConfig;
+
+export interface NgGapiClientConfig extends ClientConfig {
```

Comment:
Maybe we just can go with GapiClientConfig?

Reply:
Done